package.json

```diff
@@ -1016,6 +1016,11 @@
           ],
           "default": "off",
           "description": "Traces the communication between VS Code and the PowerShell Editor Services language server. **This setting is only meant for extension developers!**"
+        },
+        "powershell.renameSymbol.shouldGenerateAlias": {
```

Comment:
Just from a UX perspective, since this is user facing it can probably be simplified to `powershell.rename.createAlias`